update readme to remove conflicting information

coolbreeze413 opened the pull request:
the correct information is already present in https://github.com/QuickLogic-Corp/TinyFPGA-Programmer-Application/blob/master/q-series/python/readme.md#installation
Hence, remove the instructions here as they are redundant and also partially conflicting (no --recursive here)

timsaxe reviewed (changes requested):
1. I agree with ripping out the old install, but don;t like forcing the user to go to another link.  Feels like extra, unnecessary work.  Why not just incorporate the q-series readme into the primary readme?
2. Also, USB IDs may have changed.  Please check.

coolbreeze413 commented:
1. Yes you are right, it doesnt make sense to have another readme. I have updated the main readme, and modified some content to make it a bit more clearer, especially the lsusb part.
2. USB ID is still the same (2nd one 1d50:6130) so I have kept both as possibilities of how it might appear in lsusb output.

coolbreeze413 commented:
Included documentation changes as part of the update to support appfpga binary flashing, so closing this PR.
